CHANGELOG.md

```diff
@@ -11,13 +11,13 @@
 
 ### Enhancements
 
-- Added `miden-proving-service` and `miden-proving-service-client` crates (#926).
+- Added `miden-remote-prover` and `miden-remote-prover-client` crates (#926).
 - Added support for gRPC server side reflection to all components (#949).
 - Compressed faucet background image (#985).
-- Added support for TLS to `miden-proving-service-client` (#968).
+- Added support for TLS to `miden-remote-prover-client` (#968).
 - Added support for TLS to faucet's connection to node RPC (#976).
 - Replaced integer-based duration args with human-readable duration strings (#998).
-- [BREAKING] Refactor the `miden-proving-service` proxy status service to use gRPC instead of HTTP (#953).
+- [BREAKING] Refactor the `miden-remote-prover` proxy status service to use gRPC instead of HTTP (#953).
```

Comment:
The changelog shouldn't modify existing references to miden-proving-service. And should specify the change of name done by this PR (understand you are aware of this just probably overlooked these diffs given how many there are).